README.md

```diff
@@ -49,7 +49,7 @@ There are two main categories of samples: **Camera** and **Applications**. The s
 ## Instructions
 
 1. [**Install Zivid Software**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/59080712/Zivid+Software+Installation).
-Note: The samples require Zivid SDK v2 (minor version 2.2 or newer).
+Note: The samples require Zivid SDK v2 (minor version 2.3 or newer).
```

Comment:
Not needed I think. But in that case, this should work out of the box with 2.2 and 2.3. Can you test taht?

Reply:
This one is true. I guess 2.2 has already been tested? I've tested for 2.3

Reply:
Yes, 2.2 was tested. Good.